data/json/items/containers.json

```diff
@@ -916,16 +917,18 @@
     ],
     "armor_data": {
       "sided": true,
+
       "armor": [
         {
           "encumbrance": 2,
           "coverage": 5,
           "covers": [ "leg_l", "leg_r" ],
-          "material": [ { "type": "plastic", "portion_cover": 100, "thickness": 2.0 } ]
+          "specifically_covers": [ "leg_hip_r", "leg_hip_l" ]
```

Comment:
```suggestion
          "specifically_covers": [ "leg_hip_r", "leg_hip_l" ],
```

Reply:
Was going to handle the merge for you, but you un-clicked allow edits.

Reply:
Sorry not sure how that happened I've been on mobile so maybe something weird happened with that